linera-core/src/updater.rs

```diff
@@ -433,14 +486,13 @@ where
                 );
                 let query = ChainInfoQuery::new(chain_id);
                 self.remote_node.handle_chain_info_query(query).await?
```

Comment:
This is a follow-up to our discussion – we don't have the block at the specified height so how can we update the remote node about this chain. Maybe we should just leave the logging here? If we don't have the block, we won't be able to send certificates down below (b/c we don't have `target_block_height`, we probably don't have `target_block_height-1` either.

Reply:
I agree, but IIRC this made a test fail, and it might be orthogonal to this PR, so I'd rather do it in a follow-up.